src/platform/lumin-runtime/types/event-data/event-data.js

```diff
@@ -3,9 +3,11 @@
 export class EventData {
     constructor(nativeEvent) {
         this._nativeEvent = nativeEvent;
+        this._propertyNames = [];
     }
 
     _addGetProperties(propertyNames) {
+        this._propertyNames = propertyNames;
```

Comment:
shouldn't this come after you filter?

Reply:
Too much time had passed for me to remember why it was done this way. The reason is that the _nativeEvent used to filter does not exist when we are generating type script. Instead I'm just passing in a dummy that prevents a crash when we construct the wrappers: ` const dummyEvent = {
    getUiNode() {
      return {};
    }
  };`

Since this is far from obvious, I'll add a comment to clarify